content/Engineering Hiring Process.md

```diff
@@ -0,0 +1,84 @@
+# The Engineering Hiring Process at Rootstrap
+
+To be globally recognized digital masters, that’s our vision at Rootstrap. And we are convinced that in order to get there we should hire the right people, those that will help us create remarkable products, offer a premium service and will make of our office a creative and relaxed environment to work in.
+
+At Rootstrap, we believe that having a hiring process in place is the ultimate tool to help your organization find the perfect candidates to become part of your team. It should help detect those people who would fit not just technically but also those who despite having different personalities would contribute to the company's culture and diversity. That’s why everything starts with the hiring process.
+
+## How We Organize
+
+We have a hiring team of about 14 people, between HR analysts and engineers. Among the engineers, we divide ourselves depending on the technologies we specialize in. The team meets on a monthly basis and we cover subjects such as upcoming job postings, new exercises for the technical part of the interview, and in general to see what we are doing right and the things we should improve. 
+
+We see each member of this team as an ambassador to Rootstrap. Someone that can make the interviewee feel comfortable through the interview despite how nerve-wracking a process like this can be, and someone that can clearly convey Rootstrap’s culture.
+
+## Process
+
+We’re going to focus on explaining how we implement this process here at Rootstrap: how it looks like, what are the steps, which areas of the company participate in each of those steps, etc.
+
+We have four main steps. Though this may seem quite long, it’s not, a normal flow should take not much longer than 10 days. We try to maximize the interviewee’s experience while getting the most in terms of quality and information from these instances.
```

Comment:
```suggestion
We have four main steps. Though this may seem quite long, it’s not, a normal flow should not take much longer than 10 days. We try to maximize the interviewee’s experience while still getting the most out of it in terms of quality and information.
```